python/tvm/script/builder/builder.py

```diff
@@ -49,13 +49,13 @@ def get(self) -> Frame:
 DefType = TypeVar("DefType", bound=Object)
 
 
-def def_(name: str, var: DefType) -> DefType:
-    return _ffi_api.Def(name, var)  # pylint: disable=no-member # type: ignore
+def name_obj(name: str, var: DefType) -> DefType:
```

Comment:
just use `name` as a verb

```suggestion
def name(name: str, var: DefType) -> DefType:
```